external/src/components/ui3/DragAndDropDiv.tsx

```diff
@@ -53,7 +53,11 @@ export function DragAndDropDiv({
 
   if (disabled) {
     return (
-      <div ref={forwardRef} {...otherProps}>
+      <div
+        ref={forwardRef}
+        className={cx(classes.dndContainer, className)}
```

Comment:
Following the pattern in the codebase: we keep `dndContainer` class too, so if any styles are applied to it, they aren't lost when enabling/disabling file attachments